src/main/xerus/monstercat/api/Covers.kt

```diff
@@ -16,13 +16,16 @@ object Covers {
 	private fun coverCacheFile(coverUrl: String, size: Int): File {
 		coverCacheDir.mkdirs()
 		val newFile = coverCacheDir.resolve(coverUrl.substringBeforeLast('/').substringAfterLast('/').replaceIllegalFileChars())
-		return coverCacheDir.resolve("${newFile.nameWithoutExtension}x$size.${newFile.extension}")
+		return coverCacheDir.resolve("${newFile.nameWithoutExtension}x$size.jpg") // FIXME : Do not hardcode extension
```

Comment:
Why don't you use the extension of `newFile` anymore? If we don't know the extension, I would prefer to give the file no extension at all. If a reader is so versatile that it can recognize a png disguised as jpg, it can also handle files without extension. Besides, the cache is not for browsing anyways.

Reply:
The extensions returned empty each time, because ``coverUrl`` (the HTTP url to the cover) doesn't have an extension anymore. Only some gibberish by AWS...

Let's remove the hardcode then and keep files without extensions.

Reply:
Reverted in 14d34f5ed31afc8f75c85274ebcc398b8246d276